src/Features/Core/Portable/Completion/MatchResult.cs

```diff
@@ -5,13 +5,12 @@
 using System.Collections.Generic;
 using Microsoft.CodeAnalysis.PatternMatching;
 using Roslyn.Utilities;
-using RoslynCompletionItem = Microsoft.CodeAnalysis.Completion.CompletionItem;
 
 namespace Microsoft.CodeAnalysis.Completion
 {
-    internal readonly struct MatchResult<TEditorCompletionItem>
+    internal readonly struct MatchResult
     {
-        public readonly RoslynCompletionItem RoslynCompletionItem;
+        public readonly CompletionItem CompletionItem;
```

Comment:
We don't actually need to keep a reference to editor item anymore. It can be found using `IndexInOriginalSortedOrder` isntead.